specification/semantic_conventions.md

```diff
@@ -162,24 +165,19 @@ the top of the stack provided first and the bottom of the stack provided last.
 The format of each stack trace line is:
 
 ```
-        at <namespace>.<function>(<file>:<lineno>)
-        at <namespace>.<function>(<file>:<lineno> <col>)
-        at <namespace>.<function>(<file>:<lineno>:<lineno> <col>:<col>)
+  at <function>(<file>:<lineno>)
+  at <function>(<file>:<lineno>:<col>)
 ```
 
-- OPTIONAL leading whitespace
+- OPTIONAL leading whitespace (tabs or spaces)
 - OPTIONAL "at "
-- `<namespace>` - REQUIRED. Fully qualified namespace of the function call. For object-oriented
-  languages this SHOULD be the module/package and class name. If the function being executed is globally 
-  scoped this MUST be set to `global`. If class is unknown, the value MUST be set to `unknown`.
-- `<function>` - Name of the function or class method being invoked
+- `<function>` - Fully qualified name, i.e. containing the namespace, module and/or class name if applicable, of the function or method being invoked.
```

Comment:
The `function` should be REQUIRED right? Not sure why it wasn't originally.

Reply:
Thanks! Added it